<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="employes">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>1341</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/imggg.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EQUIPEMENTS">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
    </property>
    <property name="text">
     <string>Equipements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EMPLOYES">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
    </property>
    <property name="text">
     <string>Employes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="COMMANDES">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>380</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
    </property>
    <property name="text">
     <string>Cmmandes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ALIMENTATION">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>450</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
    </property>
    <property name="text">
     <string>Alimentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ANIMAUX">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>520</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 18pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
    </property>
    <property name="text">
     <string>Animaux</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="metiers">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>841</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="CRUD">
     <attribute name="title">
      <string>Crud</string>
     </attribute>
     <widget class="QTabWidget" name="crud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>841</width>
        <height>521</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>kj l </string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="liste">
       <attribute name="title">
        <string>Liste</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>0</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 24pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Employes</string>
        </property>
       </widget>
       <widget class="QLabel" name="back1_4">
        <property name="geometry">
         <rect>
          <x>-280</x>
          <y>-340</y>
          <width>1381</width>
          <height>1171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="CHERCHER">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="chercher">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>290</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Chercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="CHER">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>290</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix2/img.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableView" name="aff3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>831</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QTableView" name="aff">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>811</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="TRIER">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>290</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <zorder>back1_4</zorder>
       <zorder>label</zorder>
       <zorder>CHERCHER</zorder>
       <zorder>chercher</zorder>
       <zorder>CHER</zorder>
       <zorder>aff3</zorder>
       <zorder>aff</zorder>
       <zorder>TRIER</zorder>
      </widget>
      <widget class="QWidget" name="ajouter">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="ajt_emp">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 23pt &quot;Playbill&quot;;

</string>
        </property>
        <property name="text">
         <string>Nouvel Employe</string>
        </property>
       </widget>
       <widget class="QLabel" name="ID">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>110</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="PRENOM1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>240</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="TEL1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>240</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="AGE">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>180</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="etat">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>300</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 21pt &quot;Playbill&quot;;

</string>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ETAT1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>300</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="email">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>300</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="prenom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AJOUTER1">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>400</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="tel">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>230</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Telephone</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="EMAIL1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>300</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="age">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>170</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QLabel" name="Nom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font:19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NOM1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>180</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID1">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>110</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
       <widget class="QLabel" name="back1_2">
        <property name="geometry">
         <rect>
          <x>-270</x>
          <y>-370</y>
          <width>1381</width>
          <height>1171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="miseajour">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>400</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Mettre à jour</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SUPPRIMER1">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>100</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 15pt &quot;Playbill&quot;;
background-color: rgb(218, 178, 133);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <zorder>back1_2</zorder>
       <zorder>ajt_emp</zorder>
       <zorder>TEL1</zorder>
       <zorder>AGE</zorder>
       <zorder>etat</zorder>
       <zorder>ETAT1</zorder>
       <zorder>AJOUTER1</zorder>
       <zorder>tel</zorder>
       <zorder>age</zorder>
       <zorder>ID</zorder>
       <zorder>ID1</zorder>
       <zorder>NOM1</zorder>
       <zorder>Nom</zorder>
       <zorder>prenom</zorder>
       <zorder>email</zorder>
       <zorder>EMAIL1</zorder>
       <zorder>PRENOM1</zorder>
       <zorder>miseajour</zorder>
       <zorder>SUPPRIMER1</zorder>
      </widget>
      <widget class="QWidget" name="STATISTIQUE">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <widget class="QLabel" name="back1">
        <property name="geometry">
         <rect>
          <x>-280</x>
          <y>-370</y>
          <width>1381</width>
          <height>1171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>351</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 23pt &quot;Playbill&quot;;

</string>
        </property>
        <property name="text">
         <string>Statistiques d'âge des employes</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>501</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 178, 133);</string>
        </property>
       </widget>
       <widget class="QLabel" name="STAT">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>491</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

font: 19pt &quot;Playbill&quot;;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="statistique">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QLabel" name="back1_5">
      <property name="geometry">
       <rect>
        <x>-460</x>
        <y>-450</y>
        <width>1751</width>
        <height>1371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="trier">
     <attribute name="title">
      <string>Trie</string>
     </attribute>
     <widget class="QLabel" name="back1_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>1751</width>
        <height>1371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="back1_7">
      <property name="geometry">
       <rect>
        <x>-570</x>
        <y>-390</y>
        <width>1971</width>
        <height>1271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/imgeee.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>employes</zorder>
   <zorder>metiers</zorder>
   <zorder>EQUIPEMENTS</zorder>
   <zorder>EMPLOYES</zorder>
   <zorder>COMMANDES</zorder>
   <zorder>ALIMENTATION</zorder>
   <zorder>ANIMAUX</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
